db/structure.sql

```diff
@@ -2465,7 +2473,9 @@ ALTER TABLE ONLY public.winners
 SET search_path TO "$user", public;
 
 INSERT INTO "schema_migrations" (version) VALUES
+(20250204131151),
 (20250202220815),
+(20250130144731),
```

Comment:
looks like this line could be breaking the migration since it was removed from your other PR
```suggestion
```